wdfn-server/waterdata/templates/macros/components.html

```diff
@@ -2,8 +2,12 @@
     <div class="wdfn-component" data-component="hydrograph" data-siteno="{{ site_data.site_no }}"
          data-agency-cd="{{ site_data.agency_cd }}" data-sitename="{{ site_data.station_nm }}"
          data-parameter-code="{{ default_parameter_code }}">
-        <div class="loading-indicator-container"></div>
-        <div class="graph-container"></div>
+        <div id="hydrograph-date-controls-container"></div>
+        <div id="hydrograph-method-picker-container"></div>
+        <div id="hydrograph-wrapper">
```

Comment:
So adding the wrapper with the hydrograph and loading indicator inside allowed me to easily overlay the loading indicator over the graph.

Reply:
I was thinking that might be an option, might work for a text overlay if the graph shows up completely blank as well. Yeah, this is a great idea.